### PR TITLE
Module Upload Coursework and Retrieve Integration

### PR DESCRIPTION
**Pull request by Adam-Ellgood**
Myself and Aaron have met up to bring both of the previous pull requests (File upload/ file download) to make it integrated functionality. The functionalities are now working together within one GUI application. The functionalities now require being integrated with the login functionality, which I believe is Mustafa.

Once this process takes place, there will be 3 x integrated functionality as a team. We have 6 members in our team, it would be great to have some more integrated functionality. We can do this team! The time is becoming limited before we meet with our client again on December 15th. 

This pull request can be closed once the team feel that these are correctly integrated with one another.

### CODE REVIEW

**Comment by aaronreay** [comment]
Hi Adam, 

Very productive session this afternoon integrating our functionality together. I have downloaded the file locally Adam and it is all working perfect. We need to just test it with a student logged in, so we need to wait until tomorrow to integrate with Mustafa.

**Comment by aaronreay** [comment]
Mustafa has now integrated his functionality (login) to our upload and download functionality.

Our system now has the ability for a student to login, and have download and upload capabilities.

Everything is coming together nicely now, and hopefully we can continue to add extra functionality from our group members.

Regards,

Aaron

**Comment by Adam-Ellgood** [comment]
Nice, as this was stand alone code away from the master application, the remaining thing remaining, of which has now been complete in today daily stand up was integrating these 2 functionalities with the login functionality to the main database.

Great work!

Consider this pull request closed!

File: Module_Upload.java [hunk 1]
```diff
@@ -0,0 +1,465 @@
...
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import javax.swing.JOptionPane;
+
```
Comment (Adam-Ellgood) [inline Module_Upload.java:23]: Adam and Aaron's integrated file upload and download functionality

**Pull request closed without merging**